.github/workflow_scripts/test_tabular.sh

```diff
@@ -17,7 +17,7 @@ install_vision
 cd tabular/
 if [ -n "$ADDITIONAL_TEST_ARGS" ]
 then
-    python3 -m pytest --junitxml=results.xml --runslow "$ADDITIONAL_TEST_ARGS" tests
+    python3 -m pytest -n=2 --junitxml=results.xml --runslow "$ADDITIONAL_TEST_ARGS" tests
```

Comment:
Can we just do `-n auto`

Reply:
I think that might be a bit aggressive, since we might unexpectedly get OOMs. Shall we start with two, and move to more aggressive settings later?

Reply:
Fine with starting with 2 but would be good to controlled it in a centralized place so only minimal change required going forward. One idea is to define a new function at https://github.com/autogluon/autogluon/blob/9f33fcb7c0baebd8c9eedc1a44ebfaeb52b9518a/.github/workflow_scripts/env_setup.sh#L10 to set up an env variable `AG_TEST_PARALLELIZATION`

Reply:
That's a good point.